src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADD = 'bookstore/books/ADD';
+const REMOVE = 'bookstore/books/REMOVE';
+
+export default function booksReducer(state = [], action = {}) {
+  switch (action.type) {
+    case ADD:
+      return action.data;
+    case REMOVE:
+      return state.filter((book) => book.id !== action.data.id);
```

Comment:
- [ ] **[OPTIONAL]** From the components end of the implementation, I think we would be filtering with the id alone, instead of the entire book, so we would not be needing `action.data.id`. Instead, we can use `action.data` where `data` would be the id